packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -0,0 +1,287 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import {SafeERC20, IERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import "./libs/Errors.sol";
+import {UniversalTokenLib} from "./libs/UniversalToken.sol";
+
+import {Admin} from "./Admin.sol";
+import {IFastBridge} from "./interfaces/IFastBridge.sol";
+import {IFastBridgeV2} from "./interfaces/IFastBridgeV2.sol";
+
+contract FastBridgeV2 is IFastBridge, IFastBridgeV2, Admin {
```

Comment:
Two things here:
- `IFastBridgeV2` already extends `IFastBridge`, so there's no need to do the same for `FastBridgeV2`
  - The inheritdoc feature will still work with this change (as long as IFastBridge import exists above).
- As a rule of thumb, we try to put the interfaces at the end of the inheritance list. This doesn't affect the resulted contract, but results in a more readable structure
  - OpenZeppelin repo for reference: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/6e224307b44bc4bd0cb60d408844e028cfa3e485/contracts/governance/Governor.sol#L27
```suggestion
contract FastBridgeV2 is Admin, IFastBridgeV2 {
```